assume/reinforcement_learning/algorithms/__init__.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from torch import nn
+
+from assume.reinforcement_learning.network_architecture import MLPActor, LSTMActor
+
+policy_aliases: dict[str, type[nn.Module]] = {
```

Comment:
fair, chaged it

Reply:
renamed the "policy_class" as well into "neural_network_architecture_class", so that its clear that it's referring to the same thing (but it's a class object instead of the string/alias). I'm not too happy with the long name though. Would you prefer policy_class although the naming wouldn't be fully consistent? 

Another possibility would be: "actor_architecture" (e.g. "mlp"/"lstm"), "actor_class" (-> MLPActor/LSTMActor) and "actor_aliases" to match it. I think it would be also clearer that the config option only changes the neural network architecture of the actor, but not of the critics.

Reply:
yes, the architecture itself could be used. however, it would be necessary to implement another critic (child) class with the same architecture.

Reply:
Kim and I agreed to change the naming to actor_architecture - analog critic_architecture would be possible, but is currently not implemented